runtime/jailer_integ_test.go

```diff
@@ -74,18 +73,43 @@ func testJailer(t *testing.T, jailerConfig *proto.JailerConfig) {
 	image, err := alpineImage(ctx, client, defaultSnapshotterName)
 	require.NoError(err, "failed to get alpine image")
 
-	vmID := testNameToVMID(t.Name())
+	fcClient, err := newFCControlClient(containerdSockPath)
+	require.NoError(err)
 
+	vmID := testNameToVMID(t.Name())
+	// additionalDrive is a regular file;
+	// and the following additionalBlockDevice is a block device, aks block special file;
+	// They are used to test different bind mount cases
 	additionalDrive := internal.CreateFSImg(ctx, t, "ext4", internal.FSImgFile{
 		Subpath:  "dir/hello",
 		Contents: "additional drive\n",
 	})
 
+	var additionalBlockDevice string
+	if jailerConfig != nil {
+		// cover "With Jailer" & "With Jailer and bind-mount" test cases
+		additionalBlockDevice = internal.CreateBlockDevice(ctx, t, "ext4", internal.Devicefile{
+			Subpath: "dev/sda",
+			UID:     int(jailerConfig.UID),
+			GID:     int(jailerConfig.GID),
+			Dev:     259, // 259 is the major number for blkext which is a block device
+		})
+	} else {
+		// cover "Without Jailer" test case
+		additionalBlockDevice = internal.CreateBlockDevice(ctx, t, "ext4", internal.Devicefile{
+			Subpath: "dev/sda",
+			UID:     0, // Without jailer, set the block device user & group owners to root
+			GID:     0,
+			Dev:     259, // 259 is the major number for blkext which is one type of block devices
+		})
```

Comment:
How about declarating internalDeviceFile first and setting UID/GID later in `if`?

Reply:
Done.